code/game/atoms_movable.dm

```diff
@@ -48,6 +48,10 @@
 	///only the last container of a client eye has this list assuming no movement since SSparallax's last fire
 	var/list/client_mobs_in_contents
 
+	/// String representing the spatial grid groups we want to be held in
+	/// Key'd to a global list of lists. We do it like this to prevent people trying to mutate them
```

Comment:
```suggestion
	/// String representing the spatial grid groups we want to be held in.
	/// acts as a key to the list of spatial grid contents types we exist in via SSspatial_grid.spatial_grid_categories.
	/// We do it like this to prevent people trying to mutate them and to save memory on holding the lists ourselves
```